SwiftyContextMenu/ContextMenuViewController.swift

```diff
@@ -14,24 +14,25 @@ protocol ContextMenuViewControllerDelegate: class {
 
 class ContextMenuViewController: UIViewController {
 
-    private var contextMenu: ContextMenu!
+    private let contextMenu: ContextMenu
     private weak var delegate: ContextMenuViewControllerDelegate?
 
-    private var blurView: UIVisualEffectView = UIVisualEffectView(effect: UIBlurEffect(style: .light))
-    private var overlayView = UIView(frame: .zero)
-    private var snapshotImageView = UIImageView(frame: .zero)
-    private var contextMenuTableView = ContextMenuTableView()
-    private var contextMenuView = UIView()
+    private let blurView: UIVisualEffectView
```

Comment:
I think the type of this property should be `ContextMenuBackgroundBlurView`

Reply:
Well done 👍